x/rewards/types/iprpc.go

```diff
@@ -13,3 +13,13 @@ const (
 	// IprpcRewardsCurrentPrefix is the prefix to retrieve all IprpcRewardsCurrent
 	IprpcRewardsCurrentPrefix = "IprpcRewardsCurrent/"
 )
+
+type IprpcMemo struct {
```

Comment:
this should be defined in a proto buff

Reply:
done. see https://github.com/lavanet/lava/commit/1723bb796860c079c1c6a6e659dbe8b849c2c815